documentation/partner_backend.md

```diff
@@ -226,15 +238,22 @@ To obtain a token from the Latch Auth0 App, the following steps need to be execu
 	* `expires_in`: Time-to-live of the access token, in seconds. After this amount of time, the given access token will no longer be valid.
 	* `token_type`: Auth scheme to use (will always be "Bearer").
 
-	In case of an error, the API will return an HTTP 500 and a status in the "error_description" field:
+	In case of an error, the API will return the following error codes:
+
+	* HTTP 401
+		* `error="access_denied"`: missing or invalid credentials.
+
+			⇒ Check the request is using the correct value for `client_id` and `client_secret`.
 	
-	* `UNAUTHORIZED_API`: missing or invalid "audience" parameter.
+	* HTTP 403
+		* `error="access_denied"`: missing or invalid "audience" parameter.
 
 		⇒ Check the API request to make sure the "audience" field has the right value.
 
-	* `INTERNAL_SERVER_ERROR`: there was an unexpected error.
+	* HTTP 500
+		* `error="internal_server_error"`: there was an unexpected error.
 
-		⇒ Contact Latch Support to help debug this issue.
+			⇒ Contact Latch Support to help debug this issue.
```

Comment:
There is also a 400 Bad Request thrown when the request payload is missing the `grant_type` field:
```
{
    "error": "invalid_request",
    "error_description": "Missing required parameter: grant_type"
}
```
Lets add this scenario as well.

Reply:
great find, thanks! Done.